Daytime-Operations/Main-Telescope/MT-Calibrations/ComCam-Combined-Calibrations-Procedure/comcam-combined-calibrations-procedure.rst

```diff
@@ -167,7 +172,56 @@ After loading the script, a window that contains two sections, ``SCHEMA`` (top)
 - ``oods_timeout``: Timeout value, in seconds, for the Observatory Operations Data Service (``OODS``).
   Default: ``120``.
 
-An example set of configuration parameters is as follows:
+
+Configuration examples
+-----------------------
+
+**Preferred daily script mode to be run**: if no configuration parameters are passed to LOVE and the default parameters are used, the script will take 21 biases, 21 darks of 5 seconds each one, and 21 flats of 5 seconds each one. In each case, the first image will be discarded. New combined calibrations will not be generated, and verification of the images taken will be performed using the existing combined calibrations in the ``LSSTComCam/calib`` collection (i.e., th script will do ``external verification``). In this case, no defects will be made.  Following DMTN-222, a gain estimate will be produced from each of the 10 flat pairs taken. **Users should adjust parameters when needed, for example, the exposure times or the number of exposures taken**.
+
+If the exposure times need to change, it can be done as follows:
+
+.. code-block:: text
+    exp_times_dark: 20
+    exp_times_flats: 30
+
+If both the number of exposures and exposure times need to change, it can be done like this:
+
+.. code-block:: text
+
+    n_bias: 30
+    n_dark: 5
+    exp_times_dark: [5, 10, 15, 20, 25]
+    n_flat: 10
+    exp_times_flat: [5, 10, 15, 20, 25, 30, 35, 40, 45, 50]
+
+Example of a configuration file for ``internal_verification``. Note that the newly-generated combined calibrations
+will be certified in the ``calib_collection`` collection, so this parameter must be specified, and new validity ranges shoudl be provided (spannign one day for daily calibrations). The name of the collection needs to be changed if the script needs to be run again (or the valifdity range), as it is not possible to certify the same type of callibration in the same collection with teh same validity range:
```

Comment:
More typos: shoudl -> should, spannign -> spanning, valifdity -> validity, callibration -> calibration, teh -> the.